src/Symfony/Component/Security/Core/Tests/Authorization/Voter/AbstractVoterTest.php

```diff
@@ -65,16 +65,6 @@ public function testVoteLegacy(array $attributes, $expectedVote, $object, $messa
 
         $this->assertEquals($expectedVote, $voter->vote($this->token, $object, $attributes), $message);
     }
-
-    /**
-     * @group legacy
-     * @expectedException \BadMethodCallException
-     */
-    public function testNoOverriddenMethodsThrowsException()
-    {
-        $voter = new AbstractVoterTest_NothingImplementedVoter();
-        $voter->vote($this->token, new \stdClass(), array('EDIT'));
-    }
```

Comment:
this is because the supported class in ``AbstractVoterTest_NothingImplementedVoter`` is wrong too, same bug than in ``AbstractVoterTest_LegacyVoter`` (this is the issue when refactoring tests between branches, and not finishing things properly after the merge to newer branches)